### DISCUSSION TITLE
Navigation tutorial scene in Spoke

### DISCUSSION THREAD

**Discussion by Kanishk-Malick** [mozilla/hubs#2804] [General Questions & Discussion] [answered]
I want to create a scene in spoke where the user doesn't have any option other than moving forward, then step left or turn left then right and so on..this is basically to get used to moving around in Hubs. Are there any such scene already there ? If not, please suggest how to do it ?

**Comment by emclaren** [comment, marked as answer]
There are a few tutorial scenes in the scene database (you can find this in Hubs, if you go to switch the scene in a room and do a search for "tutorial"). I believe this was the original https://hubs.mozilla.com/scenes/LnaxgSd/tutorial-room, but the folks who made it later divided it into 3 separate tutorials, one for VR users, one for mobile users and one for desktop users. It doesn't prevent you from moving in the wrong direction, but does give some helpful information to guide you. 

I also have a folder where I add screencaptures of decorations that the community has used to share information about how to move around in Hubs, you can see them here: https://drive.google.com/drive/folders/1_6hyStLhhPSAQ8Z93dyNpWZPCzFyxJWM?usp=sharing

We are currently working on implementing a new design for the built-in Hubs tutorial, but I'm not sure when this will be ready. If you put something together, let us know, I'm sure other folks in the community would find this helpful!

**Reply by Kanishk-Malick** [reply]
@emclaren Is it possible to get the spoke link to this tutorial room ? I will make a copy of that and modify to suit our needs and share it back with the community.

**Reply by emclaren** [reply]
My apologies for the delay - I didn't see this notification in Github. To make a copy of the tutorial that you can edit in spoke, click "remix in spoke" 

<img width="634" alt="Screen Shot 2020-09-25 at 3 08 07 PM" src="https://user-images.githubusercontent.com/6835324/94320131-0d573e80-ff41-11ea-89e7-10a6fe75ad57.png">

we would love to see if you were able to come up with something new!
